integration-cli/docker_cli_service_logs_test.go

```diff
@@ -88,7 +88,7 @@ func (s *DockerSwarmSuite) TestServiceLogsCompleteness(c *check.C) {
 	lines := strings.Split(strings.TrimSpace(out), "\n")
 
 	// i have heard anecdotal reports that logs may come back from the engine
-	// mis-ordered. if this tests fails, consider the possibility that that
+	// mis-ordered. if this tests fails, consider the possibility that
```

Comment:
I think this one was correct; "the possibility that that might be occurring". The second "that" is referring to the first sentence.